StartOutDev/startout-ui
# UI pages

This PR creates a new "How you can help" page in startout_ui repository.

**TESTING**

1. Pull in the ui_pages branch.
2. Navigate to https://www.figma.com/file/G9m44YBiaFfda99X0oa19k/STAOUT20190801_DigitalAssets?node-id=107%3A0 and refer to How you can help page layout. 
3. Check for all the blocks and layout of the page.
